openshift-install-powervs

```diff
@@ -631,7 +631,9 @@ function powervs_login {
   debug_switch
   CRN=$($CLI_PATH pi service-list | grep "${SERVICE_INSTANCE_ID}" | awk '{print $1}')
   [[ -z $CRN ]] && error "Cannot find PowerVS service instance with ID: $SERVICE_INSTANCE_ID for this account"
-  $CLI_PATH pi service-target "$CRN"
+  SVCNAME=$($CLI_PATH pi service-list | grep "${SERVICE_INSTANCE_ID}" | awk '{$1=""; print $0}')
```

Comment:
`awk '{$1=""; print $0}'` gives additional space between arguments. Can we do `awk '{print $2}'` instead?

Reply:
Yes.. that additional space is a limitation, need to live with it. Cannot use $2 because name of service can have space and hence $2 will be the first word of the service name not the whole name

Reply:
Can you please trim the leading space? The log statement would not look good.

Reply:
> Can you please trim the leading space? The log statement would not look good.

done